base_attachment_object_storage/models/ir_attachment.py

```diff
@@ -40,6 +40,19 @@ class IrAttachment(models.Model):
 
     _local_fields = ('image_small', 'image_medium', 'web_icon_data')
 
+    datas = fields.Binary(
+        compute='_compute_datas',
+        inverse='_inverse_datas',
+        string='File Content',
+        nodrop=True,
```

Comment:
curious: where this param comes from? :thinking:

Reply:
@simahawk sorry didn't see that. It seems that it is so the column doesn't get droped when there is a function field with the same name. Which the original datas field is. also it was already in attachment_s3 :smiling_imp: